src/reducers/badges/view-type-reducer.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2019 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import
+{
+    RECEIVE_VIEW_TYPE,
+    RESET_VIEW_TYPE_FORM,
+    UPDATE_VIEW_TYPE,
+    VIEW_TYPE_UPDATED,
+    VIEW_TYPE_ADDED } from '../../actions/badge-actions';
+
+import { LOGOUT_USER, VALIDATE } from 'openstack-uicore-foundation/lib/actions';
+import { SET_CURRENT_SUMMIT } from '../../actions/summit-actions';
+
+export const DEFAULT_ENTITY = {
+    id                  : 0,
+    name                : '',
+    description         : '',
+    is_default          : null,
+}
+
+const DEFAULT_STATE = {
+    entity      : DEFAULT_ENTITY,
+    errors      : {}
+};
+
+const viewTypeReducer = (state = DEFAULT_STATE, action) => {
+    const { type, payload } = action
+    switch (type) {
+        case LOGOUT_USER: {
```

Comment:
u need to attend also SET_CURRENT_SUMMIT
check https://github.com/fntechgit/summit-admin/pull/88